project/Build.scala

```diff
@@ -33,15 +33,14 @@ object MinimalBuild extends Build {
     resolvers += typesafeSnapshot,
     resolvers += sonatypeSnapshot,
     // publishArtifact in packageDoc := false,
-    crossScalaVersions := Seq("2.10.4","2.11.5"),
-    scalaVersion := "2.11.5",
+    scalaVersion := "2.11.7",
```

Comment:
Thats what I thought. There is already a pull request I have submitted with your changes included:

https://github.com/romix/akka-kryo-serialization/pull/57

 Apparently, the upstream inactive for over 10 months now, and project is not being maintained anymore. 

It would be a good idea to take it over.